tests/e2e/plugins/index.js

```diff
@@ -1,7 +1,7 @@
-const webpack = require('@cypress/webpack-preprocessor');
+const webpack = require("@cypress/webpack-preprocessor");
```

Comment:
Can you format this file? I don't know why the single quotes (`'`) got converted into double ones (`"`)